<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Programmkopf</class>
 <widget class="QDialog" name="Dialog_Programmkopf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Programmkopf</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>370</y>
     <width>321</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton_Abbrechen">
      <property name="text">
       <string>Abbrechen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_save">
      <property name="text">
       <string>Speichern</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_OK">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>411</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_laenge">
        <property name="text">
         <string>Laenge</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_breite">
        <property name="text">
         <string>Breite</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_dicke">
        <property name="text">
         <string>Dicke</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_kommentar1">
        <property name="text">
         <string>Kommentar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_sicherheitsabstand">
        <property name="text">
         <string>Sicherheitsabstand</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_bezeichnung">
        <property name="text">
         <string>Bezeichnung</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Ausfuehrbedingung">
        <property name="text">
         <string>Ausfuehrbedingung</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_laenge_var">
        <property name="text">
         <string>*[L]</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_breite_var">
        <property name="text">
         <string>*[B]</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_dicke_var">
        <property name="text">
         <string>*[D]</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_kommentar1_var">
        <property name="text">
         <string>[KOM]</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label__sicherheitsabstand_var">
        <property name="text">
         <string>[SIA]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label__bezeichnung_var">
        <property name="text">
         <string>[BEZ]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>[AFB]</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit_laenge">
        <property name="frame">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_breite"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_dicke"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_kommentar1"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_sicherheitsabstand"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit__bezeichnung"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_AFB"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_laenge</tabstop>
  <tabstop>lineEdit_breite</tabstop>
  <tabstop>lineEdit_dicke</tabstop>
  <tabstop>lineEdit_kommentar1</tabstop>
  <tabstop>lineEdit_sicherheitsabstand</tabstop>
  <tabstop>lineEdit__bezeichnung</tabstop>
  <tabstop>lineEdit_AFB</tabstop>
  <tabstop>pushButton_Abbrechen</tabstop>
  <tabstop>pushButton_save</tabstop>
  <tabstop>pushButton_OK</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
